### ISSUE TITLE
Example data

### ISSUE THREAD

**Issue by haesleinhuepf** [MMV-Lab/vessel-express-napari#45]
Hi all,

just a proposal/feature request: If you provide potential users of this plugin an example dataset, they can try out the plugin on data it was made for. This gives them a feeling of success before they try it with their own data, the plugin might not be made for 😉

Thanks!

Best,
Robert

**Comment by jxchen01** [comment]
yes!!! Very good suggestion. We do have these data and actually a lot of data. They are on Zenodo. We will update the documentation accordingly. Thanks for the suggestion!
